packages/storybook/src/stories/ino-dialog/ino-dialog.stories.ts

```diff
@@ -77,18 +76,65 @@ export const Playground: Story<Components.InoDialog> = (args) => html`
         <ino-segment-button dense value="6">Summer</ino-segment-button>
       </ino-segment-group>
     </div>
-    <div class="ino-dialog-footer" slot="footer">
+    <div class="ino-dialog-footer">
       <ino-button
         icon="remove"
-        id="close-dialog-btn"
-        style="--button-color-primary: red; --button-color-primary-light: rgba(255, 0, 0, 0.7); --button-color-primary-dark: red"
+        data-ino-dialog-action="cancel"
+        fill="outline"
+        color-scheme="secondary"
         >Cancel
       </ino-button>
-      <ino-button icon="mail" id="close-dialog-btn">Submit</ino-button>
+      <ino-button icon="mail" data-ino-dialog-action="submit">Submit</ino-button>
     </div>
-  </ino-dialog>
 `;
 
+export const Playground: Story<Components.InoDialog> = (args) => html`
```

Comment:
There seems to be a problem with the playground.
Whenever I try to use the knobs (e.g. `fullwidth="true"`), I only get to see the default story.
I've checked the DOM and it seems like the `<ino-dialog>` elements are duplicated whenever the knobs are used.

How to reproduce:
1. Run the storybook
2. Go to Dialog Story
3. Press the Button `Open Dialog`
4. Close the Dialog
5. Set `fullwidth` to `true` by using the knobs
6. Press the Button `Open Dialog` again
7. See the default dialog instead of a full-width one

Reply:
Seems to be fixed with `attach-to`, even though I'm not sure why :D Let me know if it works for you as well.